banyand/liaison/grpc/property.go

```diff
@@ -762,3 +784,38 @@ type repairInProcessKey struct {
 	entity  string
 	modTime int64
 }
+
+// sortPropertiesWithSortedValues sorts properties using pre-extracted sortedValue bytes.
+func sortPropertiesWithSortedValues(
```

Comment:
To optimize efficiency in the merging process, consider implementing merge sorting, which can leverage already-sorted sub-property lists. 

For more information on the merge sort algorithm, visit the [Wikipedia page](https://en.wikipedia.org/wiki/Merge_sort).